js/options/saturationValue.js

```diff
@@ -0,0 +1,39 @@
+window.options.saturationValue = {
+  name: 'saturation value',
+  state: {
+    saturation: 50
+  },
+  run: function () {
+    const saturation = window.options.saturationValue.state.saturation
+
+    const div = document.createElement('div')
+    div.innerHTML = `
+      <style>
+        .stroke-width {
+          display: flex;
+          align-items: center;
+          margin: 0 10px;
+        }
+
+        .stroke-width > span {
+          margin-right: 10px;
+          width: 38px;
+        }
+      </style>
+
+      <div class="stroke-width">
+        <span>${saturation}</span>
+        <input type="range" min="1" max="100" value="${saturation}">
+      </div>
```

Comment:
You should rename these classes from "stroke-width" to something like "saturation-level", in part for clarity but also to avoid potential conflicts with the other stroke option window in the future. at the moment these two diff option windows (ur new saturation value && my other stroke width option) are never present at the same time so this works fine (technically speaking) but if the design changes && we give the user the ability to open up multiple option windows, then these two CSS classes will be present simultaneously && one will over-ride the other.